substrate/frame/child-bounties/src/lib.rs

```diff
@@ -207,8 +221,14 @@ pub mod pallet {
 
 	/// The description of each child-bounty.
 	#[pallet::storage]
-	pub type ChildBountyDescriptions<T: Config> =
-		StorageMap<_, Twox64Concat, BountyIndex, BoundedVec<u8, T::MaximumReasonLength>>;
+	pub type ChildBountyDescriptions<T: Config> = StorageDoubleMap<
```

Comment:
check this - https://github.com/paritytech/polkadot-sdk/pull/5961#discussion_r1827657394
lets rename the storage item